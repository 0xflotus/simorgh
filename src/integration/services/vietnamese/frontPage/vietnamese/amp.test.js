/**
 * WARNING: DO NOT EDIT. This file is automatically generated by src/integration/utils/runTests/writeTestFiles/writeTestFile.js
 * @service vietnamese
 * @pathname /vietnamese
 */

import runAmpTests from '../../../../pages/frontPage/ampTests';

describe('AMP vietnamese frontPage', runAmpTests);
