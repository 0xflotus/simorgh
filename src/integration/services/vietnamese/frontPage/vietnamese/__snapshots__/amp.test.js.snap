// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP vietnamese frontPage A11y I can see the skip to content link 1`] = `"Bỏ qua để xem nội dung"`;

exports[`AMP vietnamese frontPage AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
  "policy": Object {
    "default": Object {
      "timeout": Object {
        "fallbackAction": "reject",
        "seconds": 0,
      },
      "waitFor": Object {
        "user-consent": Array [],
      },
    },
  },
}
`;

exports[`AMP vietnamese frontPage AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP vietnamese frontPage Analytics ATI 1`] = `
Object {
  "requests": Object {
    "base": "https://logws1363.ati-host.net?",
    "pageview": "\${base}s=598343&s2=97&p=vietnamese.page&r=\${screenWidth}x\${screenHeight}x\${screenColorDepth}&re=\${availableScreenWidth}x\${availableScreenHeight}&hl=\${timestamp}&lng=\${browserLanguage}&x1=[urn:bbc:cps:3df2f586-7a4e-791d-e050-380a4e804347]&x2=[amp]&x3=[news-vietnamese]&x4=[vi]&x5=[\${sourceUrl}]&x6=[\${documentReferrer}]&x7=[index-home]&x8=[simorgh]&x9=[Tin+chính+-+BBC+News+Tiếng+Việt]&x11=[1970-01-18T01:55:33.134Z]&x12=[1970-01-19T02:08:48.910Z]&ref=\${documentReferrer}",
  },
  "transport": Object {
    "beacon": false,
    "image": true,
    "xhrpost": false,
  },
  "triggers": Object {
    "trackPageview": Object {
      "on": "visible",
      "request": "pageview",
    },
  },
}
`;

exports[`AMP vietnamese frontPage Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/vietnamese"`;

exports[`AMP vietnamese frontPage Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. BBC không chịu trách nhiệm nội dung các trang ngoài. Tìm hiểu cách chúng tôi tiếp cận việc dẫn tới trang ngoài"`;

exports[`AMP vietnamese frontPage Header I can see the branding 1`] = `"BBC News, Tiếng Việt"`;

exports[`AMP vietnamese frontPage I can see Index Alsos 1`] = `"Tin liên quan"`;

exports[`AMP vietnamese frontPage I can see sections 1`] = `"Tin chính"`;

exports[`AMP vietnamese frontPage I can see sections 2`] = `"Chuyên Đề"`;

exports[`AMP vietnamese frontPage I can see sections 3`] = `"Nghe/XemXem tất cả"`;

exports[`AMP vietnamese frontPage I can see sections 4`] = `"Xã hội"`;

exports[`AMP vietnamese frontPage I can see sections 5`] = `"Pháp luật"`;

exports[`AMP vietnamese frontPage I can see sections 6`] = `"Nhân vật"`;

exports[`AMP vietnamese frontPage I can see sections 7`] = `"Lịch sửXem tất cả"`;

exports[`AMP vietnamese frontPage I can see sections 8`] = `"EU-Việt NamXem tất cả"`;

exports[`AMP vietnamese frontPage I can see sections 9`] = `"Trung Quốc"`;

exports[`AMP vietnamese frontPage I can see sections 10`] = `"Hình ảnh"`;

exports[`AMP vietnamese frontPage I can see sections 11`] = `"Góc nhìn và chuyên mục"`;

exports[`AMP vietnamese frontPage I can see sections 12`] = `"Học viện Báo chí BBCXem tất cả"`;

exports[`AMP vietnamese frontPage I can see sections 13`] = `"Xem World Cup 2018 cùng BBC"`;

exports[`AMP vietnamese frontPage I can see the headline 1`] = `"BBC News, Tiếng Việt - Tin chính"`;

exports[`AMP vietnamese frontPage SEO Canonical link 1`] = `"http://localhost:7080/vietnamese"`;

exports[`AMP vietnamese frontPage SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP vietnamese frontPage SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP vietnamese frontPage SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP vietnamese frontPage SEO Heading level 1 1`] = `"BBC News, Tiếng Việt - Tin chính"`;

exports[`AMP vietnamese frontPage SEO Lang attribute 1`] = `"vi"`;

exports[`AMP vietnamese frontPage SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "WebPage",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/vietnamese.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/vietnamese",
    "@type": "WebPage",
    "name": "Tin chính",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/vietnamese.png",
      "width": 1024,
    },
    "name": "BBC News Tiếng Việt",
    "publishingPrinciples": "https://www.bbc.com/vietnamese/institutional-49283563",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/vietnamese.png",
  "url": "http://localhost:7080/vietnamese",
}
`;

exports[`AMP vietnamese frontPage SEO OG description 1`] = `"BBC Tiếng Việt đem lại tin tức trung thực, khách quan về thế giới và Việt Nam."`;

exports[`AMP vietnamese frontPage SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/vietnamese.png"`;

exports[`AMP vietnamese frontPage SEO OG image alt 1`] = `"BBC News Tiếng Việt"`;

exports[`AMP vietnamese frontPage SEO OG locale 1`] = `"vi-VN"`;

exports[`AMP vietnamese frontPage SEO OG site name 1`] = `"BBC News Tiếng Việt"`;

exports[`AMP vietnamese frontPage SEO OG title 1`] = `"Tin chính - BBC News Tiếng Việt"`;

exports[`AMP vietnamese frontPage SEO OG type 1`] = `"website"`;

exports[`AMP vietnamese frontPage SEO OG url 1`] = `"http://localhost:7080/vietnamese"`;

exports[`AMP vietnamese frontPage SEO Page title 1`] = `"Tin chính - BBC News Tiếng Việt"`;

exports[`AMP vietnamese frontPage SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP vietnamese frontPage SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP vietnamese frontPage SEO Twitter creator 1`] = `"@bbcvietnamese"`;

exports[`AMP vietnamese frontPage SEO Twitter description 1`] = `"BBC Tiếng Việt đem lại tin tức trung thực, khách quan về thế giới và Việt Nam."`;

exports[`AMP vietnamese frontPage SEO Twitter image alt 1`] = `"BBC News Tiếng Việt"`;

exports[`AMP vietnamese frontPage SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/vietnamese.png"`;

exports[`AMP vietnamese frontPage SEO Twitter site 1`] = `"@bbcvietnamese"`;

exports[`AMP vietnamese frontPage SEO Twitter title 1`] = `"Tin chính - BBC News Tiếng Việt"`;

exports[`AMP vietnamese frontPage Story Promo I can see a headline 1`] = `"Tân hiến pháp Bắc Hàn nói ông Kim là nguyên thủ"`;

exports[`AMP vietnamese frontPage Story Promo I can see an image 1`] = `
<amp-img
  alt="Kim Jong-un"
  attribution="KCNA"
  height="549"
  layout="responsive"
  src="https://ichef.bbci.co.uk/news/660/cpsprodpb/0D5E/production/_107222430_gettyimages-585818210.jpg"
  width="976"
/>
`;
